lib/galaxy/objectstore/irods.py

```diff
@@ -213,18 +219,20 @@ def __init__(self, config, config_dict):
         if irods is None:
             raise Exception(IRODS_IMPORT_MESSAGE)
 
-        self.session = iRODSSession(host=self.host, port=self.port, user=self.username, password=self.password, zone=self.zone)
+        self.session = iRODSSession(host=self.host, port=self.port, user=self.username, password=self.password, zone=self.zone, refresh_time=self.refresh_time)
         # Set connection timeout
         self.session.connection_timeout = self.timeout
-        log.debug("irods __init__ %s", reload_timer)
+        log.debug("irods_pt __init__: %s", ipt_timer)
```

Comment:
What is that you're measuring in this timer ? That code runs just once, I don't think it is useful to log here.